### DISCUSSION TITLE
My GitHub profile has already been approved, but the Highlight Pro badge still hasn't appeared.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#133257] [GitHub Education] [answered]
### Hi everyone,

My GitHub profile has already been approved, but the Highlight Pro badge still hasn't appeared.
![WhatsApp Image 2024-07-23 at 18 18 11_1f4aa0c6](https://github.com/user-attachments/assets/6804c6da-84b0-449c-9aeb-23b4e7e235c3)

**Comment by davevad93** [comment, marked as answer]
Hi @rubypoddar  ,

The message says you have been approved but your benefits will be available soon, that's why you don't have the `PRO` badge yet. It can take between a few days to a couple of weeks for the benefits to become available; the waiting time may vary depending on different factors, such as the volume of applications.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @rubypoddar was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470

**Reply by ghost** [reply]
Okay, then I'll wait. Thank you.😊

**Reply by davevad93** [reply]
You're welcome @rubypoddar !
